### DISCUSSION TITLE
可否考虑增加上传文件夹功能。

### DISCUSSION THREAD

**Discussion by iwvw** [AlistGo/alist#538] [Ideas]
先感谢大佬的辛苦付出💕。
想上传一个文件夹，用RaiDrive挂载webdav总是出错，想问问能不能增加一个文件夹上传入口。

**Comment by xhofe** [comment]
暂不考虑，上传通过服务器中转，所以不推荐主力使用本程序网页端上传，网页端的上传仅为临时应急使用，请优先使用原存储的官方上传。

**Reply by iwvw** [reply]
好的，谢谢大佬

**Reply by qingmuhy** [reply]
但其实考虑到crypt功能，还是最好有个上传文件夹的API。

**Reply by ypq123456789** [reply]
确实

**Comment by KnIfER** [comment]
阿里云的网页端上传挺快的，安卓也能用：全选一个文件夹内容就能上传了。
